app/src/main/res/layout/fragment_listed_memes.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.listed_memes.ListedMemesFragment">
```

Comment:
do not use **_** to separate words in packages, use **.**

Reply:
done, thanks